docs/source/RunQemuUnittests.rst

```diff
@@ -1,6 +1,48 @@
-======================================
-Running QEMU unittests with Avocado-VT
-======================================
+===========================
+Running QEMU kvm-unit-tests
+===========================
+
+Currently there are two ways to run kvm-unit-tests. Newer one with per test
+results method is `Run kvm-unit-test in avocado`_ and the older one
```

Comment:
It's hard to say. People working on kvm-unit-test tend to call it kvm-unit-test (at least from what I heard). Anyway I can change it...